typescript/src/policies/schema.ts

```diff
@@ -176,15 +266,13 @@ export const SendEvmTransactionCriteriaSchema = z
       EthValueCriterionSchema,
       EvmAddressCriterionSchema,
       EvmNetworkCriterionSchema,
+      EvmDataCriterionSchema,
     ]),
   )
   .max(10)
   .min(1);
 
-/**
- * Type representing a set of criteria for the sendEvmTransaction operation.
- * Can contain up to 10 individual criterion objects of ETH value or EVM address types.
- */
+// Type representing a set of criteria for the sendEvmTransaction operation. Can contain up to 10 individual criterion objects of ETH value or EVM address types.
```

Comment:
this comment doesn't use jsdoc syntax because of a strange ts bug that puts it in places it shouldn't be in the output, breaking type definitions.
<img width="868" alt="Screenshot 2025-06-18 at 16 53 35" src="https://github.com/user-attachments/assets/3d32751d-91c4-47df-80e6-37615482e63e" />